chapter_convolutional-modern/batch-norm.md

```diff
@@ -434,13 +442,13 @@ class BatchNorm(tf.keras.layers.Layer):
             mean, variance = batch_mean, batch_variance
         else:
             mean, variance = self.moving_mean, self.moving_variance
-        output = tf.nn.batch_normalization(
+        output = batch_norm(
             inputs,
-            mean=mean,
-            variance=variance,
-            offset=self.beta,
-            scale=self.gamma,
-            variance_epsilon=1e-5)
+            moving_mean=mean,
+            moving_var=variance,
+            beta=self.beta,
+            gamma=self.gamma,
+            eps=1e-5)
```

Comment:
Generally we want to keep coding block short so it saves space. Can you be greedy with line breaks when listing arguments?

Reply:
It also applied to other places.

Reply:
Thanks. Fixed.